src/bridging/BridgingSdk/getErc20Decimals.ts

```diff
@@ -0,0 +1,14 @@
+import { Provider } from '@ethersproject/providers'
+import { Contract } from '@ethersproject/contracts'
+import { Signer } from '@ethersproject/abstract-signer'
+import { GetErc20Decimals } from '../types'
+import { TargetChainId } from '../../chains'
+
+const ERC20_DECIMALS_OF_ABI = ['function decimals() external view returns (uint8)'] as const
```

Comment:
Nitpick: sounds a bit weird, maybe the `OF` shouldn't be there?

```suggestion
const ERC20_DECIMALS_ABI = ['function decimals() external view returns (uint8)'] as const
```